### DISCUSSION TITLE
Cannot query data in supabase dashboard - BUG

### DISCUSSION THREAD

**Discussion by holdehn** [supabase/supabase#14457] [General]
I cannot query any data from my supabase project dashboard right now:

<img width="1330" alt="Screenshot 2023-05-18 at 11 12 41 AM" src="https://github.com/supabase/supabase/assets/109185265/bf52955b-5598-41b5-a30f-c954ab981d38">
<img width="1338" alt="Screenshot 2023-05-18 at 11 35 46 AM" src="https://github.com/supabase/supabase/assets/109185265/b18038e2-0472-44c0-805e-c8f83de83409">

Any ideas?

**Comment by alerodrod** [comment]
Same for me with logs explorer today.

**Reply by holdehn** [reply]
Weird, they told me it wasn't a priority because I am not on a paid plan but hopefully they fix it soon.
